### PR TITLE
Chunks syntax highlighting

### PR DESCRIPTION
**Pull request by TimvdLippe**
image

Run `bower update` to fetch the correct element.

### CODE REVIEW

**Comment by eanker** [comment]
Can you state the file from where the diff is located.

File: src/pull-request/pull-request-chunk.html [hunk 1]
```diff
@@ -14,22 +20,23 @@
         overflow: hidden;
       }
       .diff {
-        height: 50px;
+        border-bottom: black 1px solid;
```
Comment (eanker) [inline src/pull-request/pull-request-chunk.html:22]: Do we want this black line to separate diffs in a chunk?

Comment (TimvdLippe) [inline src/pull-request/pull-request-chunk.html:22]: No, this is for debugging purposes only. @txsmith will fix this :heart:

**Comment by eanker** [comment]
LGTM

**Pull request merged**